### ISSUE TITLE
npm install is broken by pngquant-bin dependency

### ISSUE THREAD

**Issue by anthonyaam** [Klathmon/imagemin-webpack-plugin#56]
I'm getting the below error when running `npm install imagemin-webpack-plugin`. 

`pngquant failed to build, make sure that libpng is installed`

[This pngquant-bin issue describes the problem in the pngquant-bin dependency](https://github.com/imagemin/pngquant-bin/issues/78).

**Comment by Klathmon** [comment]
Unfortunately this is a problem with pngquant and not something I can fix.

However, I did just update an old branch named `no-default-plugins` that you can use as a workaround until pngquant fixes their stuff.

You should include it in your package.json like this:

    "imagemin-webpack-plugin": "Klathmon/imagemin-webpack-plugin#no-default-plugins"

You will need to manually install any plugins you want yourself and put their options in the `plugins` option in this webpack plugin in order for anything to be optimized/compressed.

**Comment by anthonyaam** [comment]
Even though the code fix will be outside of imagemin-webpack-plugin, I would suggest keeping this issue open while users are unable to `npm install imagemin-webpack-plugin` so that other users can find these instructions.

**Comment by Klathmon** [comment]
Fair enough! I won't really be checking back on it all that often though, so if someone notices that this is fixed ping me here so I can close this and update anything that's needed.

**Comment by codejet** [comment]
Version 6.0.0 of imagemin-pngquant was released. Maybe it fixes this issue? If so, updating the dependency would be nice (I guess a PR for that would be overkill...?).

**Comment by Klathmon** [comment]
I'll update it later today and see if it solves the problem, thanks for the heads up!

And a PR is never overkill!

**Comment by Klathmon** [comment]
@anthonyaam version 2.2.0 of this plugin was published that upgrades imagemin to version 6. Can you give it a try at some point and see if it solves your issues? Thanks!

**Comment by GitGump** [comment]
@Klathmon I change imagemin-webpack-plugin version to 2.2.0, and npm install, still it trys to install pngquant-bin@5.0.1 and failed, I don't know why.
